src/IntegralArrays.jl

```diff
@@ -1,5 +1,78 @@
 module IntegralArrays
 
 # package code goes here
+using Images, ClosedIntervals
```

Comment:
Considering that order switching and emptiness checking is the issue, what would be the way to go? Should IntervalSets be derived from UnitRanges?